sympy/solvers/polysys.py

```diff
@@ -27,8 +27,14 @@ def solve_poly_system(seq, *gens, **args):
     gens: generators
         generators of the equations in seq for which we want the
         solutions
+    strict: a boolean (default is False)
+        if strict is True, NotImplementedError will be raised if
+        the solution is known to be incomplete (which can occur if
+        all solutions are not expressible in radicals)
```

Comment:
There is a difference between "all solutions are not" and "not all solutions are". Here it should be the latter.